Gaijinizer patch/Enemies.txt

```diff
@@ -528,12 +526,11 @@ It will be difficult to get out of it any longer.#MTLed
 だんだんと精神を蝕まれていく。
 愛する人を想う強固な意志をも崩されてしまうほどに。
 > CONTEXT: Enemies/503/BookDescription/3
-Barghest's magic is concentrated in the tongue and vagina.
-When she is complimented or committed, she is definitely
-attracted
-And the spirit will be saved.
-To the extent that even a strong will that deems a loved
-one is broken.#MTLed
+A barghest's magic is concentrated in her tongue and
+vagina. If you're licked or raped by her, you will
+become charmed. And your spirit will gradually be
+undermined. This magic is enough to even break the
+will of those who strongly love another.#Ron
```

Comment:
Here's what I'm getting out of the translation of the third line which makes a bit more sense:
>_**If you're licked or are being raped by her, you will become charmed, quickly breaking your spirit and resistance.**_